### DISCUSSION TITLE
Does Self Hosted version get regular holiday updates from the main public API?

### DISCUSSION THREAD

**Discussion by cdevelin** [nager/Nager.Date#666] [Q&A] [answered]
I'm interested in the self hosted licensing model, but it doesn't mention whether or how/when the self-hosted version can get holiday updates from the regularly updated public API.

Is there some form of periodic scheduled synchronization process?

**Comment by tinohager** [comment, marked as answer]
It is a docker container that can be easily updated via latest tag
